model/license.go

```diff
@@ -262,6 +273,17 @@ func (l *License) ToJson() string {
 	return string(b)
 }
 
+func (l *License) IsTrialLicense() bool {
+	return l.IsTrial || (l.ExpiresAt-l.StartsAt) <= trialDuration.Milliseconds()
+}
+
+func (l *License) IsSanctionedTrial() bool {
```

Comment:
Add quick unit test for this function.

Reply:
Done. Added missing tests for `IsTrialLicense()` as well.